backend-services/user-verification-service/src/main/java/org/cardano/foundation/voting/domain/entity/DiscordUserVerification.java

```diff
@@ -32,9 +33,14 @@ public class DiscordUserVerification extends AbstractTimestampEntity {
     @Setter
     private String eventId;
 
-    @Column(name = "stake_address")
+    @Column(name = "wallet_id")
     @Nullable
-    private String stakeAddress;
+    private String walletId;
+
+    @Column(name = "wallet_id_type")
```

Comment:
Why call it, wallet_id_type? Shouldn't it be called wallet_type?

Reply:
Yes, make sense. I just updated it to `wallet_type`